cmake/modules/static-analysis.cmake

```
@@ -22,35 +23,50 @@ find_program(CPPCHECK cppcheck)
 find_program(CPPCHECK_HTMLREPORT cppcheck-htmlreport)
 
 if(NOT CPPCHECK)
-  message(STATUS "cppcheck command not found, static code analysis using cppcheck will not be available.")
+	message(
+		STATUS
+			"cppcheck command not found, static code analysis using cppcheck will not be available."
```

I would align it to `STATUS`
```suggestion
		"cppcheck command not found, static code analysis using cppcheck will not be available."
```